src/compiler/typescript_to_rust/generate_import_symbol_table/debug.ts

```diff
@@ -0,0 +1,88 @@
+import { SymbolTables, SymbolTable } from '../../utils/types';
```

Comment:
Please comment that this should be removed once extra credit and other functionality is finished